resources/entrypoint.sh

```diff
@@ -25,40 +25,45 @@ print(client.decrypt(CiphertextBlob=binary_data)["Plaintext"].decode())
 ' $1
 }
 
-if [[ -n "$GOOGLE_CLIENT_SECRET" ]]; then
-    client_secret=$GOOGLE_CLIENT_SECRET
-elif [[ -n "$GOOGLE_CLIENT_SECRET_KMS_BLOB" ]]; then
-    client_secret=$(kms_decrypt $GOOGLE_CLIENT_SECRET_KMS_BLOB)
-else
-    echo "Either GOOGLE_CLIENT_SECRET or GOOGLE_CLIENT_SECRET_KMS_BLOB must be set."
-    exit 1
-fi
+if [[ -z "$AUTH_HEADER_NAME" ]]; then
```

Comment:
Rather than an `if/else`, I'd suggest a `if/elseif/else` here. Explicitly error out if either of these things are not set.